### DISCUSSION TITLE
Obsidian web for multiple vaults?

### DISCUSSION THREAD

**Discussion by julie-za** [coddingtonbear/obsidian-web#204] [Q&A] [answered]
I am struggling to see how can I configure the settings to work with two different vaults? I am working on two different ones simultaneously now, but from the options in the chrome extension I understand that once you change the API URL/Key, then the notes are redirected to the new vault.

Any help on this? Anyone has ideas how to create a workflow around this issue?

(I did my best to search the discussions, but in vain. Sorry if this is a repeated question)

**Comment by coddingtonbear** [comment, marked as answer]
I'm afraid that although Obsidian Web can connect to vaults that are running Obsidian Local REST API on any port now, we still don't have any utilities in place for letting you easily switch between vaults.

**Reply by julie-za** [reply]
Thanks for letting me know!
